components/textInputWithComplete.tsx

```diff
@@ -24,12 +27,13 @@ const TextInputWithCompletion: FC<
   const handleFocus = (e: any) => {
     setVisible(true);
   };
-  const handleBLur = async (e: any) => {
-    let __ = await sleep(250);
-    setVisible(false);
+  const handleBLur = (e: any) => {
+    props.onOwnBlur?.(e.target.value);
+    // let __ = await sleep(250);
+    // setVisible(false);
     field.onBlur(e);
   };
-  let shouldShow = visible && field.value.trim().length > 0;
+  // let shouldShow = visible && field.value.trim().length > 0;
```

Comment:
Actually I commented almost all the code from this component as it's currently useless. I prefer keep it in case we want to refactor it when we'll work on having autocomplete again